modules/ROOT/pages/mule-upgrade-tool.adoc

```diff
@@ -427,6 +384,31 @@ The Mule upgrade tool requires both the current Mule instance and the new Mule i
 
 Perform a manual upgrade to ensure that the current and new Mule instances are in version 4.1.5 or later before attempting to upgrade using the Mule upgrade tool.
 
+[[upgrade_clustered_mule_instances]]
+== Upgrading Clustered Mule Instances
+
+The Mule upgrade tool does not automatically support upgrading clustered Mule instances. To upgrade clustered Mule instances, you have to manually upgrade each node using the tool.
+
+=== Before You Begin
+
+Before starting your upgrade, ensure that you have:
```

Comment:
"... ensure that you have:"
is followed by an imperative bullet point "* Copy ..." 

Should it be "* A copy of" or should "ensure that you have:" change to something else that flows into the imperative?

Reply:
Modifying the first bullet to "a copy of" so as not to modify the include.